recipe/meta.yaml

```diff
@@ -1,18 +1,18 @@
 {% set name = "holidays" %}
-{% set version = "0.18" %}
+{% set version = "0.29" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: e94d9d3536ff1a2a70d6d2f56cb055eb9a95d0b9a3a5fb81b9e8f9d36e333789
+  sha256: e8219df1570dd92f17244ff7da93f57630b0dd2fedf86e86b4110f07825b0a67
 
 build:
-  skip: True     # [py<37]
```

Comment:
so is this really not necessary anymore, even though the upstream clearly states a restriction on python version?
`python_requires = >=3.8`
in this case it would be `skip: True  # [py<38]`